templates/network/dns.erb

```diff
@@ -2,9 +2,9 @@
 <% for @ip in @nameservers -%>
 nameserver <%= @ip %>
 <% end -%>
-<% for @domain in @domains -%>
-search <%= @domain %>
-<% end -%>
-<% for @option in @options -%>
-options <%= @option %>
-<% end -%>
+<%- unless @domains.empty? -%>
+  search <%= @domains.join(' ') -%>
```

Comment:
It's nice to indent the erb, but won't that come out odd in the resolved template?

Reply:
You can use ```<%=  "search #{domains.join(' ') -%>```

Reply:
Somehow the whitespace isn't rendering properly there, but extra spaces between the ```=``` and ```"```